src/common/components/vanilla/button/button-positive.css

```diff
@@ -0,0 +1,29 @@
+$button-color: $success;
```

Comment:
@bartaz: A separate folder makes it marginally easier to extract into a separate folder later.

My feeling with the buttons is to have a different component for every named component in Vanilla. The difference is marginal, it's a personal preference to have numerous smaller elements than fewer switched elements.